walkthroughs-config.json

```diff
@@ -1,3 +1,3 @@
 {
-    "prettyName": "Red Hat Solution Patterns (OS4)"
+    "prettyName": "Red Hat Solution Patterns-TONY (OS4)"
```

Comment:
this should be removed

Reply:
Removed with latest commit

Reply:
The "(OS4)" should also be removed. This was part of a PR that I did recently.